pkg/network/tracer/tracer_test.go

```diff
@@ -1628,12 +1646,16 @@ func TestHTTPStats(t *testing.T) {
 }
 
 func TestHTTPSViaLibraryIntegration(t *testing.T) {
+	cfg := testConfig()
+
 	if !httpSupported(t) {
 		t.Skip("HTTPS feature not available on pre 4.1.0 kernels")
 	}
-
-	if strings.HasPrefix(runtime.GOARCH, "arm") {
-		t.Skip("this feature is not yet support on arm")
+	if !httpsSupported(t) {
+		t.Skip("HTTPS feature not available on ARM pre 5.5.0 kernels")
+	}
+	if runningOnARM() && !cfg.EnableRuntimeCompiler {
```

Comment:
Can't we support prebuilt on ARM (>= 5.5.0) now that the loader does the helper translation for us?

Reply:
yes, the issue is that offset-guessing is not working properly, I will take a look tomorrow